server/manifest.go

```diff
@@ -46,11 +46,13 @@ func (m *Manifest) Remove() error {
 }
 
 func (m *Manifest) RemoveLayers() error {
-	for _, layer := range append(m.Layers, m.Config) {
-		if err := layer.Remove(); errors.Is(err, os.ErrNotExist) {
-			slog.Debug("layer does not exist", "digest", layer.Digest)
-		} else if err != nil {
-			return err
+	for _, layer := range append(m.Layers, &m.Config) {
```

Comment:
It's a little strange that m.Layers is a pointer, yet m.Config isn't. I think the better answer is to change `Layers []*Layer` to `Layers []Layer` (not sure why this was a slice of pointers in the first place) but that might be out of scope of this change and can be solved later.

Reply:
I think that is reasonable but it's not directly related to this patch so I want to break it out and do it separately in a new PR.

Reply:
Sounds good